### PR TITLE
[TASK] Overhaul Startpage and menu / toctree configuration

### PR DESCRIPTION
**Pull request by linawolf**

### CODE REVIEW

File: Documentation/GeneralConventions/FileStructure.rst [hunk 1]
```diff
@@ -71,41 +71,13 @@ displayed on GitHub and GitLab.
 .. _index-rst:
 .. _start-file:
 
-Startpage: :file:`Documentation/Index.rst`
-------------------------------------------
+Start page: :file:`Documentation/Index.rst`
+-------------------------------------------
 
 The documentation index file at :file:`Documentation/Index.rst` is the starting
```
Comment (brotkrueml) [inline Documentation/GeneralConventions/FileStructure.rst:77]: Maybe better would be: "The documentation start page is the entry point". "Index file" sounds pretty technical.

File: Documentation/GeneralConventions/FileStructure.rst [hunk 2]
```diff
@@ -71,41 +71,13 @@ displayed on GitHub and GitLab.
...
-theme, partly from Settings.cfg, partly by internal calculations. For more
-information about all available variables, see the
-:doc:`Replacements <t3sphinxtest:Replacements/Index>` chapter of the Sphinx
-theme documentation.
-
-The placeholders of pattern `<name>` must be replaced manually by the author of
-the documentation:
+further pages.
 
 The startpage should contain an anchor target :rst:`..  _start:` above its
```
Comment (brotkrueml) [inline Documentation/GeneralConventions/FileStructure.rst:82]:
```suggestion
The start page should contain an anchor target :rst:`..  _start:` above its
```

File: Documentation/GeneralConventions/FileStructure.rst [hunk 3]
```diff
@@ -114,70 +86,31 @@ document title. This way you can link to a documents start page by:
 
     See :ref:`TYPO3 Explained <t3coreapi:start>`.
 
+If your manual has more pages then this one it must contain a table of content
```
Comment (brotkrueml) [inline Documentation/GeneralConventions/FileStructure.rst:89]:
```suggestion
If your manual has more pages then this start page it must contain a table of content
```

More clear.

File: Documentation/GeneralConventions/FileStructure.rst [hunk 4]
```diff
@@ -114,70 +86,31 @@ document title. This way you can link to a documents start page by:
...
-Abstract
-^^^^^^^^
+Configure the menu - the toctree
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The *abstract* placeholder contains a short and precise description of the
-project. It should follow the abstract of :file:`README.rst` and - if
-available - the description fields of :file:`ext_emconf.php` and
-:file:`composer.json.`
+The `toctree` on the startpage can be hidden, it then only influences the main
```
Comment (brotkrueml) [inline Documentation/GeneralConventions/FileStructure.rst:99]:
```suggestion
The `toctree` on the start page can be hidden, it then only influences the main
```

File: Documentation/GeneralConventions/FileStructure.rst [hunk 5]
```diff
@@ -191,61 +124,52 @@ overview. The TOC is built with the
...
-   .
-   └── Documentation
-       └── Usage
-           └── Typo3Backend
-               └── ShopDashboard
-                   ├── Index.rst
-                   └── SellWidget.rst
+    ..  toctree::
+        :hidden:
+        :caption: BASICS
```
Comment (brotkrueml) [inline Documentation/GeneralConventions/FileStructure.rst:146]: I would avoid uppercase words. If they should be uppercase by design, that should be done by CSS. If not, it is up to the user and we should not give that example.

**Review by brotkrueml — CHANGES_REQUESTED** [review]

File: Documentation/GeneralConventions/FileStructure.rst (hunk 5 above)
Comment (linawolf) [inline Documentation/GeneralConventions/FileStructure.rst:146]:
```suggestion
        :caption: Basics
```

File: Documentation/GeneralConventions/FileStructure.rst [hunk 6]
```diff
@@ -71,113 +71,46 @@ displayed on GitHub and GitLab.
 .. _index-rst:
 .. _start-file:
 
-Startpage: :file:`Documentation/Index.rst`
-------------------------------------------
+Start page: :file:`Documentation/Index.rst`
+-------------------------------------------
 
 The documentation index file at :file:`Documentation/Index.rst` is the starting
```
Comment (linawolf) [inline Documentation/GeneralConventions/FileStructure.rst:77]:
```suggestion
The documentation start page (:file:`Documentation/Index.rst`) is the entry
```

File: Documentation/GeneralConventions/FileStructure.rst [hunk 7]
```diff
@@ -191,61 +124,52 @@ overview. The TOC is built with the
...
+        :caption: Basics
 
+        About
+        HowToGetHelp
+        ...
 
-.. _index-rst-meta-menu:
+    .. toctree::
+        :hidden:
+        :caption: HOWTOS
```
Comment (brotkrueml) [inline Documentation/GeneralConventions/FileStructure.rst:154]:
```suggestion
        :caption: Howtos
```

File: Documentation/GeneralConventions/FileStructure.rst [hunk 8]
```diff
@@ -191,61 +124,52 @@ overview. The TOC is built with the
...
 
-The *meta-menu* placeholder builds a second menu that is not visible in the
-content area of the page, but only in the left sidebar (desktop) or toggle menu
-(mobile) - separate from the main menu. It contains links to functional pages,
-such as the :ref:`Sitemap.rst <sitemap-rst>` and the
-:ref:`genindex.rst <genindex-rst>`, which do not provide content specific to
-this documentation, as follows:
+    ..  toctree::
+        :hidden:
+        :caption: ADVANCED
```
Comment (brotkrueml) [inline Documentation/GeneralConventions/FileStructure.rst:162]:
```suggestion
        :caption: Advanced
```

File: Documentation/GeneralConventions/FileStructure.rst [hunk 9]
```diff
@@ -191,61 +124,52 @@ overview. The TOC is built with the
...
+        :caption: Advanced
 
-..  code-block:: rst
+        HowToAddTranslation/Index
+        GeneralConventions/HowToUpdateDocs
+        GeneralConventions/ReviewInformation
 
     ..  toctree::
         :hidden:
+        :caption: MAINTAINERS
```
Comment (brotkrueml) [inline Documentation/GeneralConventions/FileStructure.rst:170]:
```suggestion
        :caption: Maintainers
```

**Review by brotkrueml — APPROVED** [review]

**Pull request merged**